README.md

```diff
@@ -17,3 +17,7 @@ Before you open source your project, consider the effort required to maintain it
 ## FAQ
 
 #### Who owns the code I contribute to open source projects?
+
+## License
+
+This repository is licensed under [CC-BY-4.0](../LICENSE) (c) 2019 GitHub, Inc.
```

Comment:
The perpose of this repostory is to learn how to use the opersource broject.
if you need to contact with someone, contact with me.

Thank you